src/schema/graphql-base.ts

```diff
@@ -98,6 +99,28 @@ export const DIRECTIVES: DocumentNode = gql`
     "Declares a field to be indexed"
     directive @index(sparse: Boolean = false) on FIELD_DEFINITION
 
+    "Declares a field to be indexed with ArangoSearch"
+    directive @flexSearch(includeInSearch: Boolean = false) on FIELD_DEFINITION
+
+    "Declares a field to be indexed with ArangoSearch with a Text Analyzer"
+    directive @flexSearchFulltext(language: FlexSearchLanguage, includeInSearch: Boolean = false) on FIELD_DEFINITION
+
+
+    "The available languages for ArangoSearch Analyzers"
+    enum FlexSearchLanguage {
+        EN, DE, ES, FI, FR, IT, NL, NO, PT, RU, SV, ZH
+    }
+
+    "An argument to define the order of a flexSearchIndex"
+    input FlexSearchOrderArgument {
+        field: String,
```

Comment:
forgot to check, can you specify a dot-seprated path here?

Reply:
yes you can

Reply:
hm I guess 'field' is ok, the regular index configuration also calls it 'fields' (where you can specify multiple paths). You could add a description here in graphql-base though. I don't know if the autocompletion currently uses it, but it might in the future.